doc/source/conf.py

```diff
@@ -798,3 +798,35 @@ def setup(app):
     app.add_autodocumenter(AccessorMethodDocumenter)
     app.add_autodocumenter(AccessorCallableDocumenter)
     app.add_directive("autosummary", PandasAutosummary)
+
+
+#### Ignore list for broken link checks
+
+import re
```

Comment:
This is not an ignore list. Have a look at https://github.com/pandas-dev/pandas/pull/55246#pullrequestreview-1647254547 and follow the instructions.

Reply:
I think I didn't understand the proper way of doing the ignore list. Should I take `linkcheck_ignore = [r'http://localhost:\d+/']` and put all the [broken](https://github.com/pandas-dev/pandas/actions/runs/6420191351/job/17431740319?pr=55246#step:6:9) links from the yaml file and put that list inside the conf.py file at the end ? 

What is the use of this [file](https://pipelinesghubeus22.actions.githubusercontent.com/xZyE9jtmkxWlfCAbyu1SHPJOlsa2huNFYcxohSTomy6EbdNZT9/_apis/pipelines/1/runs/531865/signedlogcontent/2?urlExpires=2023-10-10T19%3A08%3A26.1254826Z&urlSigningMethod=HMACV1&urlSignature=Zgha0BunA3OJneuIap0FFToSo%2FK3YhduNbCHSBXb%2Fv8%3D) ?

Reply:
> Should I take `linkcheck_ignore = [r'http://localhost:\d+/']` and put all the [broken](https://github.com/pandas-dev/pandas/actions/runs/6420191351/job/17431740319?pr=55246#step:6:9) links from the yaml file and put that code inside the conf.py file at the end ?

Yes. And add a comment on what this ignore list does including a link to this PR.

> What is the use of this [file](https://pipelinesghubeus22.actions.githubusercontent.com/xZyE9jtmkxWlfCAbyu1SHPJOlsa2huNFYcxohSTomy6EbdNZT9/_apis/pipelines/1/runs/531865/signedlogcontent/2?urlExpires=2023-10-10T19%3A08%3A26.1254826Z&urlSigningMethod=HMACV1&urlSignature=Zgha0BunA3OJneuIap0FFToSo%2FK3YhduNbCHSBXb%2Fv8%3D) ?

Not sure where you are getting the links, but again it is expired.